src/commands/BootstrapCommand.js

```diff
@@ -512,7 +515,7 @@ export default class BootstrapCommand extends Command {
                   "Replacing with symlink..."
                 );
                 // remove installed dependency
-                packageActions.push((cb) => FileSystemUtilities.rimraf(pkgDependencyLocation, cb));
+                packageActions.push((cb) => FileSystemUtilities.rimraf([pkgDependencyLocation], cb));
```

Comment:
We could probably batch this better, though it seems relatively low on the list of performance gains.